ES/ESdd034.xml

```diff
@@ -164,15 +172,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                               <dim type="left">5</dim>
                               <dim type="intercolumn"/>
                            </dimensions>
-                           <note>Texts I-1, I-2, I-3: all data for <locus target="#3r">3r</locus>.</note>
+                           <note>Texts <ref target="#ms_i1.1 #ms_i1.2 #ms_i1.3"/>: all data for <locus target="#3r"/>.</note>
```

Comment:
split and add type
`<ref type="item" target="#ms_i1.1>text 1</ref>, <ref  type="item" target="#ms_i1.2">text 2</ref>` etc

Reply:
not yet?